src/uwtools/logger.py

```diff
@@ -0,0 +1,151 @@
+import sys
+import functools
+from pathlib import Path
+from typing import Union, List
+import logging
+
+__all__ = ['Logger']
+
+
+class ColoredFormatter(logging.Formatter):
+    """
+    Logging colored formatter
+    adapted from https://stackoverflow.com/a/56944256/3638629
+    """
+
+    grey = '\x1b[38;21m'
```

Comment:
Suggest either extrapolate the color code into another module or create a ticket to abstract it out.

Reply:
Can do.
Since the formatter is specifically for the logger, where would you recommend putting it?

Reply:
Create a slim class called ColorLogger or something underneath that overrides the base with color options.  

Should be quick copy/paste.